recipe/meta.yaml

```diff
@@ -21,14 +21,12 @@ requirements:
   host:
     - python {{ python_min }}
     - pip
-    - poetry >=1.8
   run:
```

Comment:
Beat to me to it! I leave it to you of course...

Reply:
I still managed to get it wrong, though